src/datasets.js

```diff
@@ -373,3 +376,26 @@ export default {
             });
     },
 };
+
+export function parseResponse(response, disableBodyParser) {
+    const contentType = response.headers['content-type'];
+    const wrappedItems = wrapArray(response);
+    try {
+        if (!disableBodyParser) wrappedItems.items = Utils.parseBody(wrappedItems.items, contentType);
+    } catch (e) {
+        if (e.message.includes('Unexpected end of JSON input')) {
```

Comment:
Please add some comment why this error is retryable

Reply:
And I would call it `parseDatasetItemsResponse` to make it clear